.github/workflows/notify-release.yml

```diff
@@ -0,0 +1,16 @@
+name: Notify release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 8 * * *'
+  release:
+    types: [published]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/github-action-notify-release@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this is unnecessary, please remove (yes I know that the reference repo probably has it, but it's outdated)